eslint.config.js

```diff
@@ -0,0 +1,33 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import eslintConfigPrettier from "eslint-config-prettier";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
+import eslintPluginUnicorn from "eslint-plugin-unicorn";
+
+export default [
+  {
+    languageOptions: { globals: globals.node },
+    plugins: { unicorn: eslintPluginUnicorn },
+  },
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
+  eslintConfigPrettier,
+  eslintPluginPrettierRecommended,
+  {
+    files: ["**/*.ts"],
+    rules: {
+      "no-empty-pattern": "off",
+      "no-empty": "off",
+      "@typescript-eslint/no-empty-object-type": "off",
```

Comment:
this should be fixed
https://www.totaltypescript.com/the-empty-object-type-in-typescript

Reply:
see https://github.com/RobinTail/express-zod-api/tree/eslint-recommended

Reply:
addressed